modules/misc/src/postprocessors/InternalVolume.C

```diff
@@ -13,6 +13,9 @@ InputParameters
 validParams<InternalVolume>()
 {
   InputParameters params = validParams<SideIntegralPostprocessor>();
+  params.addClassDescription("Computes the volume of an enclosed area by calculating the integral "
```

Comment:
Computes the volume of an enclosed area by performing an integral over a user-supplied boundary